### ISSUE TITLE
page, per_page

### ISSUE THREAD

**Issue by nsulikowski** [HubertJ/QuandlCS#3]
It'd be nice to add the following two query parameters... page, per_page

**Comment by HubertJ** [comment]
Hey @nsulikowski,  

Sorry for sitting on this for nearly a year without response. Please accept my apologies. Not cool on my part. 

Shouldn't be too hard to add the two extra parameters (and any else that I have neglected over the last year) so will try to get on it this week. 

HubertJ

**Comment by nsulikowski** [comment]
Not at all! I appreciate your putting the project up in Github in the first place.
thank you!
